app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,61 @@
+class FriendshipsController < ApplicationController
+
+  def index
+    @friendships = Friendship.where(user_id: current_user.id)
+  end
+
+  def create
+
+     @friendship = Friendship.create(user_id: current_user.id, friend_id: find_friend_id(params[:email]))
+     send_notification(@friendship.user.firstname, "Invite for friendship")
+     redirect_to friendship_path(@friendship)
+  end
+
+  def new
+    @user = current_user
```

Comment:
You probably don't need @user here. If it's for the view, declare current_user a helper_method. You're not using its value here. Also, you probably shouldn't be setting current_user in a form for a new. I (user) could just change its value in chrome. You are checking current_user in create (good) so I suspect the @user here is never used.